recipes/default.rb

```diff
@@ -17,7 +17,7 @@
 # limitations under the License.
 #
 package node['rsyslog']['package_name']
-package rsyslog_relp_package if node['rsyslog']['use_relp']
+package rsyslog_relp_package if node['rsyslog']['use_relp'] && node['os'] != 'freebsd'
```

Comment:
Can you please address this delivery warning?

Reply:
I'm not sure what you mean by 'delivery warning.'  Can you elaborate?

Reply:
If you click on the `Files Changed` tab, you should see a yellow warning displaying a cookstyle warning about the use of 
`node['os]`

Reply:
This made me think more about node['os'] and I created https://github.com/chef/cookstyle/issues/842 to catch additional places where a user might use it.